webapp/components/post_view/components/post_message_view.jsx

```diff
@@ -64,7 +85,18 @@ export default class PostMessageView extends React.Component {
         return (
             <span
                 onClick={Utils.handleFormattedTextClick}
-                dangerouslySetInnerHTML={{__html: TextFormatting.formatText(this.props.message, options)}}
+                dangerouslySetInnerHTML={{
```

Comment:
@hmhealey, Done. Also, did a missing change in .post__body's style for compact view.